numpy/core/src/multiarray/alloc.c

```diff
@@ -48,11 +48,7 @@ static cache_bucket datacache[NBUCKETS];
 static cache_bucket dimcache[NBUCKETS_DIM];
 
 /* as the cache is managed in global variables verify the GIL is held */
-#if defined(NPY_PY3K)
 #define NPY_CHECK_GIL_HELD() PyGILState_Check()
```

Comment:
This is only used in two places inside this file ([see here](https://github.com/numpy/numpy/blob/950dd4e15ea0976bb671148440e036c3ae2dc11d/numpy/core/src/multiarray/alloc.c#L70)). I'm happy to inline it if people want.

Reply:
Yeah, I think that makes sense. Everything else looks good to me.

Reply:
Inlined.